cmd/operator-sdk/alpha/scorecard2/cmd.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard2
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dsl      string
+	bundle   string
+	selector string
+)
+
+//scorecard2 alpha command stub
+func NewCmd() *cobra.Command {
```

Comment:
We should call this from here: https://github.com/operator-framework/operator-sdk/blob/56ef70dc1ca6015066916a196e7eeb758a6f899c/cmd/operator-sdk/alpha/cmd.go#L29-L31